Client/Frontend/Settings/AppSettingsTableViewController.swift

```diff
@@ -173,6 +173,11 @@ class AppSettingsTableViewController: SettingsTableViewController, FeatureFlagga
             AutoCompleteSettings(prefs: prefs),
             PersonalSearchSettings(prefs: prefs)
         ]
+        
+        // Ecosia: Unleash Shortcuts Experiment
```

Comment:
Very minor, but at least to me this phrasing seems that the shortcuts are from Unleash 🙈 

How about `Quick Search Shortcuts Experiment`? For me "experiment" already points to unleash implicitly anyway.
I know this is used everywhere as well and like that, just think this is a bit more clear as to what it is 🙂

Reply:
No even the smallest strong opinion on that. Happy to rename it.

Reply:
In fact, just updated 😝